### DISCUSSION TITLE
SBrick

### DISCUSSION THREAD

**Discussion by MarlioR** [pybricks/support#702] [General]
Curious to know if pybricks software can control an sbrick ??
Thanks

**Comment by laurensvalk** [comment]
No, Pybricks currently only runs on LEGO hubs.

**Reply by MarlioR** [reply]
Thanks for the quick reply. Do you think range of hubs may increase in the future?

**Reply by laurensvalk** [reply]
It might, if there are enough users who want it, or if the hub manufacturer or others want to [sponsor it](https://github.com/sponsors/pybricks).
